src/ray/gcs/redis_module/ray_redis_module.cc

```diff
@@ -430,7 +430,7 @@ int Set_DoPublish(RedisModuleCtx *ctx, RedisModuleString **argv, bool is_add) {
   }
 }
 
-int Set_DoWrite(RedisModuleCtx *ctx, RedisModuleString **argv, int argc, bool is_add) {
+int Set_DoWrite(RedisModuleCtx *ctx, RedisModuleString **argv, int argc, bool is_add, bool &changed) {
```

Comment:
The convention we've been using is to use pointers instead of references for function outputs.

Reply:
Just learnt that google style also requires using references for output args, https://google.github.io/styleguide/cppguide.html#Reference_Arguments

Reply:
Thanks! Updated to `bool *`